.goreleaser.yaml

```diff
@@ -123,7 +123,7 @@ brews:
 
 msi:
   - id: agentuity
-    name: "agentuity-{{.MsiArch}}"
+    name: "agt-{{.MsiArch}}"
```

Comment:
we can't change the name of this binary as other scripts depend on this name. can we investigate another solution? if not, let's just remove the attestation step